customize.sh

```
@@ -40,7 +40,7 @@ sleep 0.5
 ui_print "- [14/14] Applying final optimizations for peak system performance & responsiveness... ⚙️"
 sleep 0.5
 ui_print "Now Starting SuperMario-Tweaker Script"
-${MODDIR}/SuperMario-Tweaker.sh" > /dev/null
+sh ${MODPATH}/SuperMario-Tweaker.sh" > /dev/null
```

Shellcheck (0.10.0)

[warning] Did you forget to close this double quoted string?

(SC1078)